src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -100,6 +108,7 @@ def export(
                 headers=self._headers,
             ) as resp:
                 resp.raise_for_status()
+                self.__logger.debug(resp.content)
```

Comment:
```suggestion
                self.__logger.debug("got response:" + resp.content)
```
or similar (unless the content is self-explanatory anyway)